src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+    action="/" 
+    method="get" 
+    data-qa="big"
+    class="search-form .search-form_big"
+    >
+      <label class="search-form_label search-form_label_big">
+        <input 
+      type="text" 
+      name="big-search" 
+      placeholder="Try &quot;Los Angeles&quot;" 
+      data-qa="keypress"
+      class="input input_big"
+        >
```

Comment:
Pay attention to your code style, please
```suggestion
        <input 
          type="text" 
          name="big-search" 
          placeholder="Try &quot;Los Angeles&quot;" 
          data-qa="keypress"
          class="input input_big"
        >
```